patches/mojo-public-tools-bindings-checks-mojom_attributes_check.py.patch

```diff
@@ -1,36 +1,12 @@
 diff --git a/mojo/public/tools/bindings/checks/mojom_attributes_check.py b/mojo/public/tools/bindings/checks/mojom_attributes_check.py
-index fd15dfd7f7cad25559d9796e08cf1a6e9573d1b7..ed8ce7a208e01019949ad2863332c4efb1cff10c 100644
+index fd15dfd7f7cad25559d9796e08cf1a6e9573d1b7..ab90e2c1686f13852b4cb655d85c4422e5b903ea 100644
 --- a/mojo/public/tools/bindings/checks/mojom_attributes_check.py
 +++ b/mojo/public/tools/bindings/checks/mojom_attributes_check.py
-@@ -20,6 +20,7 @@ _COMMON_FIELD_ATTRIBUTES = _COMMON_ATTRIBUTES | {
- # Note: `Default`` goes on the default _value_, not on the enum.
- # Note: [Stable] without [Extensible] is not allowed.
- _ENUM_ATTRIBUTES = _COMMON_ATTRIBUTES | {
-+    'BraveAdd',
-     'Extensible',
-     'Native',
-     'Stable',
-@@ -34,6 +35,7 @@ _ENUMVAL_ATTRIBUTES = _COMMON_ATTRIBUTES | {
- }
- 
- _INTERFACE_ATTRIBUTES = _COMMON_ATTRIBUTES | {
-+    'BraveAdd',
-     'RenamedFrom',
-     'RequireContext',
-     'ServiceSandbox',
-@@ -57,6 +59,7 @@ _MODULE_ATTRIBUTES = _COMMON_ATTRIBUTES | {
- _PARAMETER_ATTRIBUTES = _COMMON_FIELD_ATTRIBUTES
+@@ -7,6 +7,7 @@ import mojom.generate.check as check
+ import mojom.generate.module as module
  
- _STRUCT_ATTRIBUTES = _COMMON_ATTRIBUTES | {
+ _COMMON_ATTRIBUTES = {
 +    'BraveAdd',
```

Comment:
please also add `'BraveExtend'` into the list. even if it's not triggered now, it should be here.

Reply:
Fixed in a2806c4690b1d675eb4ca5adf37701a4056ecfa6.